src/SchemalessAttributesTrait.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Spatie\SchemalessAttributes;
+
+use Illuminate\Support\Collection;
+use Illuminate\Support\Str;
+
+/**
+ * @property array $schemalessAttributes
+ *
+ * @mixin Collection
+ */
+trait SchemalessAttributesTrait
+{
+    /**
+     * @var
+     */
+    private $initializeSchemalessAttributesTrait;
+
+    /**
+     * Get the casts array.
+     *
+     * @return array
+     */
+    public function getCasts()
+    {
+        if (Str::startsWith(app()->version(), '5.6')) {
```

Comment:
Instead of doing this check, let's just drop support for Laravel 5 allthogheter

Reply:
@freekmurze Done. Please check